client/navigation/stylesheets/index.scss

```diff
@@ -17,28 +17,59 @@
 		display: none !important;
 	}
 
+	#woocommerce-embedded-navigation {
+		position: fixed;
+		top: 0;
+		left: 0;
+		width: $navigation-width;
+		height: 100%;
+		box-sizing: border-box;
+		background-color: $gray-900;
+		z-index: 1100; //Must be greater than z-index on .woocommerce-layout__header
+	}
+
 	&:not(.is-folded) {
 		#wpcontent,
 		#wpfooter {
 			margin-left: $navigation-width;
+
+			@media ( max-width: 960px ) {
+				margin-left: 0;
+			}
 		}
 	}
 
 	&.is-folded {
 		#wpcontent,
 		#wpfooter {
-			margin-left: 60px;
+			margin-left: 0;
 		}
-	}
 
-	#woocommerce-embedded-navigation {
-		position: fixed;
-		top: 0;
-		left: 0;
-		width: $navigation-width;
-		height: 100%;
-		box-sizing: border-box;
-		// @todo This should be updated to G2.darkGray.primary when possible.
-		background-color: #1e1e1e;
+		.woocommerce-layout__header {
+			margin-left: $header-height;
+		}
+
+		#woocommerce-embedded-navigation {
+			width: $header-height;
+			height: $header-height;
+
+			.woocommerce-navigation-header {
+				&.is-scrolled {
+					box-shadow: 0 8px 8px 0 rgba(85, 93, 102, 0.3);
```

Comment:
A variable for this might be good so we don't get out of sync with the header should it ever change.

Reply:
Good call. Done :+1: